tmt/steps/execute/internal.py

```diff
@@ -303,6 +315,10 @@ def _run_tests(self, guest, extra_environment=None):
             if self._handle_reboot(test, guest):
                 continue
             self._results.append(self.check(test))
+            try:
+                self.check_abort_file(test)
+            except tmt.utils.AbortTestError:
+                exit_first = True
```

Comment:
I'd say tmt shouldn't use exit_first for this otherwise prints `warn: Test NAME failed, stopping execution.` and we loose information it was aborted.
AFAIK tmt should say `warn: Test NAME aborted, ...` and put note: `abort` into the results (similar to timeout note) so it is visible in report.

Reply:
I've updated the logic in my most recent commit. Now when an abort has occurred it will be logged as such.